frontend/src/components/column-selectors/single-column-selector/single-column-selector.tsx

```diff
@@ -40,9 +66,11 @@ export default function SingleColumnSelector({
       itemRenderer={renderColumnItem}
       onItemSelect={onColumnSelect}
     >
-      <Button rightIcon="double-caret-vertical" disabled={disabled === true}>
-        {selected ? selected.name : "Select a column"}
-      </Button>
+      <SingleColumnSelectorButton
+        selected={selected}
+        disabled={!!disabled}
```

Comment:
Why do we need to do this? Is it not already set? Likely indicates that we should refactor if "disabled" needs to be cast to a boolean and isn't already